src/components/agenda-page/WeekSwitcher.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { faArrowRight, faArrowLeft } from "@fortawesome/free-solid-svg-icons";
+import IconButton from "../IconButton";
+import moment from "moment";
+import { useRecoilState } from "recoil";
+import { weekState } from "../../store";
+
+const WeekSwitcher: React.FC = () => {
+  const [week, setWeek] = useRecoilState(weekState);
+  const weekTitle = React.useMemo(
+    () => moment().week(week).day(0).format("DD MMM. YYYY"),
+    [week]
+  );
+
+  const handleClick = React.useCallback(
+    (left: boolean) => {
+      left ? setWeek(week - 1) : setWeek(week + 1);
+    },
+    [week, setWeek]
+  );
+
+  return (
+    <div className="flex gap-2 items-center">
+      <IconButton icon={faArrowLeft} onClick={() => handleClick(true)} />
+      <div>
+        <span className="text-xl">Week of</span>
+        <span className="text-xl font-bold"> {weekTitle}</span>
+      </div>
+
+      <IconButton icon={faArrowRight} onClick={() => handleClick(false)} />
```

Comment:
Not sure if this is possible but adding a "no highlight" style on this would be nice so clicking doesn't highlight the text